composer.json

```diff
@@ -1,16 +1,25 @@
 {
     "name":        "hostnet/asset-bundle",
     "type":        "symfony-bundle",
-    "description": "Bundle to glue Assets",
+    "description": "Hostnet asset bundle allows for configuring the hostnet/asset-lib from within your symfony app",
     "license":     "MIT",
+    "authors": [
+        {
+            "name":  "Yannick de Lange",
+            "email": "ydelange@hostnet.nl"
+        }
+    ],
     "require": {
-        "php":               ">=7.1",
-        "hostnet/asset-lib": "dev-master",
-        "symfony/symfony":   "^3.3"
+        "php":                      "^5.6|^7.0",
```

Comment:
Current support: php 7.1

I'd prefer to not lower the support to 5.6, since it's not even actively supported anymore.
If we would change the php versions that we support, we need a build matrix + update the lib first (that requires >=7.1)

Reply:
woeps, bad copy paste